chapters/en/slides/module1/module1_01.Rmd

```diff
@@ -61,242 +64,122 @@ as we will see later.
 
 ### Can you spot the differences between these four sets of numbers?
```

Comment:
This question needs rephrasing - we can spot many differences... the exact numbers are different! Maybe the question should be "Can you see differences in trends or patterns between these four sets of numbers?"